examples/vanilla-dom-vite/vite.config.js

```diff
@@ -0,0 +1,19 @@
+import {defineConfig} from 'vite';
+import path from 'path';
+
+const packagesBase = path.join(__dirname, '..', '..', 'packages');
+
+function getPathForPackage(packageName) {
+  return path.join(packagesBase, packageName, 'src');
+}
+
+export default defineConfig({
+  root: 'app',
+  resolve: {
+    alias: {
+      '@remote-ui/core': getPathForPackage('core'),
+      '@remote-ui/rpc': getPathForPackage('rpc'),
+      '@remote-ui/dom': getPathForPackage('dom'),
+    },
```

Comment:
Vite doesn't do this automatically, based on the contents of the `tsconfig.json`?

Reply:
I haven't set up path aliases in the tsconfig. I could try and see if it fixes it.

Reply:
looks like folks usually do aliases in both. 
https://www.npmjs.com/package/vite-tsconfig-paths seems to address this, but I think it's fine for these few packages?